pandas-stubs/core/indexes/base.pyi

```diff
@@ -138,7 +138,7 @@ class Index(IndexOpsMixin, PandasObject):
     def drop_duplicates(
         self, keep: NaPosition | Literal[False] = ...
     ) -> IndexOpsMixin: ...
-    def duplicated(self, keep: _str = ...): ...
+    def duplicated(self, keep: bool | _str = ...): ...
```

Comment:
You mentioned that the docs say `‘first’, ‘last’, False`.

how about: `Literal["first", "last", False]`

Reply:
Also, can you add a test?